.github/ISSUE_TEMPLATE/package_bug.yml

```diff
@@ -3,41 +3,72 @@ description: 'Report a bug, something does not work as it supposed to'
 title: '[package] <LIBRARY-NAME>/<LIBRARY-VERSION>: SHORT DESCRIPTION'
 labels: bug
 body:
-- type: markdown
+- type: textarea
   attributes:
+    label: Description
+    description: What is not working? What were you expecting? Are there any workarounds?
     value: |
-      Please don't forget to update the issue title.
-      Include all applicable information to help us reproduce your problem.
+      _Please don't forget to update the issue title._
+      
+      Missing profile or a short log will make extremely difficult to investigate your case or provide any help,
+      please, include all applicable information with details to help us reproduce your problem.
+  validations:
+    required: true
 
 - type: textarea
   attributes:
     label: Package and Environment Details
-    description: include every applicable attribute)
+    description: (include every applicable attribute)
     value: |
      * Package Name/Version: **zlib/1.2.8**
      * Operating System+version: **Linux Ubuntu 18.04**
      * Compiler+version: **GCC 8**
      * Docker image: **conanio/gcc8**
      * Conan version: **conan 1.18.0**
      * Python version: **Python 3.7.4**
+    placeholder: |
+      cat /etc/lsb-release | grep
+      gcc --version
```

Comment:
What if a different compiler used?
Also, this information is already in a conan profile

Reply:
This is only shown if they delete all the previous text playing with it now that it's merged

The place holder can not be copy pasted so they'll need to take inspiration 🙌 😀 hopefully